desc/magnetic_fields/_core.py

```diff
@@ -592,6 +593,12 @@ def save_mgrid(
             Size to split computation into chunks of evaluation points.
             If no chunking should be done or the chunk size is the full input
             then supply ``None``. Default is ``None``.
+        source_grid : Grid
```

Comment:
Just to be clear the source_grid resolution N not only decides the number of modes that define the position of a coil, but also the number of points used to discretize it to calculate the field, right?

Reply:
It does not do anything about the number of modes. It only is used to determine the points in `zeta` (or `s` or whatever coordinate the coils are parametrized with) that the coil position and tangents isareevaluated at, to be used in the biot-savart law as the source terms.